jStorage.dropbox.js

```diff
@@ -79,6 +79,13 @@
 
             });
         },
+        listStorages: function (callback) {
```

Comment:
do we need to add this function to other providers as well? jStorage.localStorage for example.